snuba/datasets/entities/sessions.py

```diff
@@ -253,10 +264,11 @@ def __init__(self) -> None:
 
     def get_extensions(self) -> Mapping[str, QueryExtension]:
         return {
-            "timeseries": TimeSeriesExtension(
+            "timeseries": SessionsTimeSeriesExtension(
```

Comment:
Extensions are deprecated and are not being used anymore in prod. All this logic is done in the `TimeSeriesProcessor` now.

Reply:
Will do